### PR TITLE
[custom_card_scenes] Option to trigger automation

### PR DESCRIPTION
**Pull request by sildehoop**

### CODE REVIEW

**Comment by mpeterson** [comment]
@sildehoop you can discover the need to use the `automation.trigger` service by looking at the domain of the entity. That way you'll save yourself the variable.

**Comment by sildehoop** [comment]
Oh nice i will do that.

**Comment by sildehoop** [comment]
```yaml
```

**Comment by mpeterson** [comment]
@sildehoop  https://github.com/UI-Lovelace-Minimalist/UI/blob/2f9e167c9ca8541dab63933553efb706c3324c3f/custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/actions/actions_card.yaml#L10

File: custom_cards/custom_card_scenes/card_scenes.yaml [hunk 1]
```diff
@@ -10,30 +10,35 @@ card_scenes:
       icon_color: "gray"
       name: "n/a"
       bg_color: "gray"
+      automation: false
```
Comment (mpeterson) [inline custom_cards/custom_card_scenes/card_scenes.yaml:13]: Don’t forget to remove the variables which are not needed anymore

File: custom_cards/custom_card_scenes/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ hide:
 
 # Custom-card "Scenes Card"
 
-The `card_scenes` you can control up to 5 `entity` such as `scene` or `script`.
+The `card_scenes` you can control up to 5 `entity` such as `scene` , `script` or `automation`.
```
Comment (mpeterson) [inline custom_cards/custom_card_scenes/README.md:10]: Nit: you probably want to remove the space before the comma

File: custom_cards/custom_card_scenes/card_scenes.yaml [hunk 3]
```diff
@@ -58,7 +58,15 @@ card_scenes:
           color_bg: "[[[ return variables.entity_1.bg_color ]]]"
         tap_action:
           action: "call-service"
-          service: "homeassistant.turn_on"
+          service: |
+            [[[
+            let domain = variables.entity_1.entity_id.substr(0, variables.entity_1.entity_id.indexOf("."));
```
Comment (mpeterson) [inline custom_cards/custom_card_scenes/card_scenes.yaml:63]: In all blocks the let is not at the right indentation

File: custom_cards/custom_card_scenes/card_scenes.yaml [hunk 4]
```diff
@@ -118,9 +150,17 @@ card_scenes:
...
+            let domain = variables.entity_5.entity_id.substr(0, variables.entity_5.entity_id.indexOf("."));
+              if (domain == "automation") {
+                return "automation.trigger"
+              } else {
+                return "homeassistant.turn_on"
+              }
+            ]]]
           service_data:
-            entity_id: "[[[ return variables.entity_5.entity_id ]]]"
+            entity_id: "[[[ return variables.entity_1.entity_id ]]]"
```
Comment (mpeterson) [inline custom_cards/custom_card_scenes/card_scenes.yaml:163]: Entity 5, right?

**Review by mpeterson — CHANGES_REQUESTED** [review]

**Comment by sildehoop** [comment]
You are very sharp😉

**Review by mpeterson — APPROVED** [review]
LGTM

**Review by basbruss — APPROVED** [review]

**Pull request merged**